instat/dlgTransformClimatic.vb

```diff
@@ -38,35 +38,39 @@ Public Class dlgTransformClimatic
 
     Private Sub InitialiseDialog()
         ucrBase.iHelpTopicID = 358
+
         Dim dctInputSumPairs As New Dictionary(Of String, String)
+
+        ' Setting receivers
         ucrReceiverStation.Selector = ucrSelectorTransform
         ucrReceiverYear.Selector = ucrSelectorTransform
         ucrReceiverDate.Selector = ucrSelectorTransform
         ucrReceiverDOY.Selector = ucrSelectorTransform
-        ucrReceiverData.Selector = ucrSelectorTransform
-        ucrReceiverDate.SetMeAsReceiver()
+
         ucrReceiverStation.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "station" & Chr(34)})
         ucrReceiverDate.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "date" & Chr(34)})
         ucrReceiverDOY.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "doy" & Chr(34)})
         'this is not right
-        ucrReceiverData.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "rain" & Chr(34)})
         ucrReceiverYear.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "year" & Chr(34)})
         ucrReceiverStation.bAutoFill = True
         ucrReceiverDate.bAutoFill = True
         ucrReceiverDOY.bAutoFill = True
-        ucrReceiverData.bAutoFill = True
         ucrReceiverYear.bAutoFill = True
 
+
         ucrReceiverData.SetParameter(New RParameter("data", 0))
         ucrReceiverData.SetParameterIsString()
         ucrReceiverData.bWithQuotes = False
         ucrReceiverData.SetParameterIncludeArgumentName(False)
+        ucrReceiverData.Selector = ucrSelectorTransform
+        ucrReceiverData.AddIncludedMetadataProperty("Climatic_Type", {Chr(34) & "rain" & Chr(34)})
+        ucrReceiverData.bAutoFill = True

```

Comment:
Moved all of ucrReceiverData to be together.
Will do the same with the other receivers when they've got parameters set up